yew-macro/src/html_tree/html_tag/tag_attributes.rs

```diff
@@ -307,14 +307,59 @@ impl Parse for TagAttributes {
             i += 1;
         }
         let booleans = TagAttributes::drain_boolean(&mut attributes);
+        for boolean in &booleans {
+            if boolean.question_mark.is_some() {
+                return Err(syn::Error::new_spanned(
+                    &boolean.label,
+                    format!(
+                        "The '{}' attribute does not support being used as an optional attribute",
```

Comment:
```suggestion
                        "the '{}' attribute does not support being used as an optional attribute",
```

Reply:
I think this error message should be more along the lines of: "boolean attributes don't support being used as an option attribute".
The current error might give the impression that only the attribute at hand  isn't supported but in actuality no boolean attributes are.